be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.h

```diff
@@ -9,7 +9,9 @@ namespace starrocks::pipeline {
 class AggregateBlockingSinkOperator : public Operator {
 public:
     AggregateBlockingSinkOperator(int32_t id, int32_t plan_node_id, AggregatorPtr aggregator)
-            : Operator(id, "aggregate_blocking_sink_operator", plan_node_id), _aggregator(aggregator) {}
+            : Operator(id, "aggregate_blocking_sink_operator", plan_node_id), _aggregator(aggregator) {
```

Comment:
remove suffix "_operator" from names of {Aggregate, AggregateDistinct}{Blocking, Streaming}{Sink, Source}Operator

Reply:
fixed